catalog/app/utils/PackageUri.spec.js

```diff
@@ -0,0 +1,32 @@
+import * as PackageUri from './PackageUri'
+
+// TODO: more coverage
```

Comment:
I think this comment is obvious and not needed :)

Reply:
it just indicates the work to be done